x-pack/plugins/observability_ai_assistant/server/functions/esql/index.ts

```diff
@@ -208,52 +209,24 @@ export function registerEsqlFunction({
               <query>
               \`\`\`
 
-              Prefer to use commands and functions for which you have documentation.
-              
-              Pay special attention to these instructions. Not following these instructions to the tee
-              will lead to excruciating consequences for the user.
-              
-              #1
-              Directive: ONLY use aggregation functions in STATS commands, and use ONLY aggregation functions in stats commands, NOT in SORT or EVAL.
-              Rationale: Only aggregation functions are supported in STATS commands, and aggregation functions are only supported in STATS commands. 
-              Action: Create new columns using EVAL first and then aggregate over them in STATS commands. Do not use aggregation functions anywhere else, such as SORT or EVAL.
-              Example: EVAL is_failure_as_number = CASE(event.outcome == "failure", 1, 0) | STATS total_failures = SUM(is_failure_as_number) BY my_grouping_name
-
-              #2
-              Directive: Use the \`=\` operator to create new columns in STATS and EVAL, DO NOT UNDER ANY CIRCUMSTANCES use \`AS\`.
-              Rationale: The \`=\` operator is used for aliasing. Using \`AS\` leads to syntax errors.
-              Action: When creating a new column in a command, use the = operator.
-              Example: STATS total_requests = COUNT(*)
+              Prefer to use commands and functions for which you have requested documentation.
 
-              #3
-              Directive: Use placeholder values for information that is missing.
-              Rationale: It is critical to generate a syntactically valid query.
-              Action: When you don't know the arguments to a function because information is missing, use placeholder values.
-              Example: "Here's an ES|QL query that generates a timeseries of 50 buckets calculating the average duration. I've used 
-              "2023-01-23T12:15:00.000Z" and "2023-01-23T12:30:00.000Z" as placeholder values. Replace them with the start
-              and end date that work for your use case."
-
-              #4
-              Directive: Wrap string literals in double quotes.
-              Rationale: It is critical to generate a syntactically valid query.
-              Action: When using string literals in function calls, wrap them in double quotes, not single quotes.
-              Example: DATE_EXTRACT("year", @timestamp)
-              
-              At the start of every message, YOU MUST, for every instruction that is relevant to the query you want to construct,
-              repeat its directives, verbatim, at the start of every message. Exclude the rationales, actions, and examples. Follow
-              it up by using a delimiter: --
+              DO NOT UNDER ANY CIRCUMSTANCES use commands or functions that are not a capability of ESQL
```

Comment:
Should ESQL be ES|QL here? Maybe nit, maybe it matters to the LLM..?

Reply:
maybe a little, I've corrected it either way, thanks!